src/EventStore.Projections.Core/Services/Processing/ResultEventEmitter.cs

```diff
@@ -1,8 +1,10 @@
 using System;
+using Serilog;
 
 namespace EventStore.Projections.Core.Services.Processing {
 	public class ResultEventEmitter : IResultEventEmitter {
 		private readonly ProjectionNamesBuilder _namesBuilder;
+		private bool _largeStateWarningLogged = false; //This would be set to true as soon as the projection state size becomes greater than 8MB and a warning is logged, preventing flooding of logs with warnings each time an event is emitted.
```

Comment:
i think the comment isn't necessary for this now that the variable name is clear